lib/workers/repository/process/lookup/index.spec.ts

```diff
@@ -1841,16 +1841,94 @@ describe('workers/repository/process/lookup/index', () => {
       });
     });
 
-    it('handles replacements', async () => {
+    it('handles replacements - name only without pinDigests enabled', async () => {
+      config.depName = 'openjdk';
+      config.currentValue = '17.0.0';
+      config.datasource = DockerDatasource.id;
+      config.versioning = dockerVersioningId;
+      // This config is normally set when packageRules are applied
+      config.replacementName = 'eclipse-temurin';
+      docker.getReleases.mockResolvedValueOnce({
+        releases: [
+          {
+            version: '17.0.0',
```

Comment:
can you add a second greater version to be sure we don't break it in future

Reply:
test case updated